bot/code_review_bot/report/phabricator.py

```diff
@@ -50,7 +50,7 @@
 """
 
 FRONTEND_LINKS = """
-You can view these defects on [the code-review frontend]({frontend_url}) and on [Treeherder]({treeherder_url}).
+You can view these defects in the Diff Detail section of [Phabricator diff {diff_id}](?id={diff_id}), on [the code-review frontend]({frontend_url}) and on [Treeherder]({treeherder_url}).
```

Comment:
Phabricator is sending an email with this message, so we should use the full URL and not a relative one, otherwise it'll be impossible to open the link from the email